recipe/meta.yaml

```diff
@@ -23,6 +23,7 @@ requirements:
     - {{ compiler('c') }}
     - {{ compiler('cxx') }}
     - make
+    - {{ mpi }}  # [build_platform != target_platform and mpi == "openmpi"]
```

Comment:
Any explanation why we need this? Or is it just a workaround that happens to make things working for unknown reasons?

Reply:
I believe this is the standard way of cross-compiling OpenMPI applications in conda-forge.

Reply:
See e.g. 

https://github.com/search?q=org%3Aconda-forge+build_platform+%21%3D+target_platform+and+mpi+%3D%3D+%22openmpi%22&type=code 

and

 https://github.com/conda-forge/openpmd-api-feedstock/blob/84ffc17ab9be95b277ee94d238a5f58ce87307dd/recipe/meta.yaml#L64

Reply:
OK. Thanks. Look, buddy, as time passes, I understand less and less of all the conda-forge quirks. If you are confident everything you did is OK, please confirm and I'll merge this PR straight away. Should I, or should we wait for @isuruf comments?

Reply:
I think the changes are pretty straightforward and I'd vote for merging this. If there any issues, I'll open a PR to follow up.